common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerActivity.java

```diff
@@ -112,9 +146,11 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             data.putExtra(RESULT_CODE, resultCode);
 
             LocalBroadcastManager.getInstance(getApplicationContext()).sendBroadcast(data);
+        } else {
+            // This means the broker is unexpectedly killed. (tested by killing the broker process via adb).
+            returnsExceptionOnActivityUnexpectedlyKilled();
```

Comment:
but the result is still received? I see it being set to true above

Reply:
yes. this function will be triggered, but the result code will be 0, and the intent will be null.

Reply:
I suppose it just depends on when it is unexpectedly killed. I would think there would be cases where it is unexpectedly killed and also a response is NOT received.